setup.py

```diff
@@ -103,6 +103,7 @@ def read_file(fname):
             "tk-docs-preview = tk_toolchain.cmd_line_tools.tk_docs_generation:main",
             "tk-run-app = tk_toolchain.cmd_line_tools.tk_run_app:main",
             "tk-config-update = tk_toolchain.cmd_line_tools.tk_config_update:main",
+            "tk-build-resource = tk_toolchain.cmd_line_tools.tk_build_resource:main",
```

Comment:
I'd suggest `tk-build-resources` (with s).

Reply:
Yes I was also thinking of having "qt" somewhere in the name. But also small is beautiful and easier to use .... so I'm not sure.